scripts/uploads/merge_and_uncode_rca_uploads.R

```diff
@@ -698,90 +685,147 @@ save_output_synapse <- function(cohort) {
   }
 }
 
-# synapse login -------------------
+# main ----------------------------
 
-status <- synLogin(auth = auth)
+main <- function(){
+
+  cohorts <- names(config$upload)
+  cohorts_str <- paste0(cohorts, collapse = ", ")
+
+  option_list <- list( 
+    make_option(c("-c", "--cohort"), type = "character",
+                help=glue("BPC cohort (choices: {cohorts_str}, {config$misc$all})")),
+    make_option(c("-u", "--save_synapse"), action="store_true", default = FALSE, 
+                help="Save output to Synapse"),
+    make_option(c("-a", "--synapse_auth"), type = "character", default = NA,
+                help="Path to .synapseConfig file or Synapse PAT (default: normal synapse login behavior)"),
```

Comment:
Nit: don't need to do this now, but in the future if you see these, it can all be removed since we added support SYNAPSE_AUTH_TOKEN style login to the synapse client.